addon-entwicklung.md

```diff
@@ -1 +1,169 @@
 # AddOn Entwicklung
+
+* [Minimale Erfordernisse](#anker-minimal)
+* [package.yml](#anker-package-yml)
+  * [Erforderliche Angabe](#anker-yaml-required)
+  * [Optionale Angabe](#anker-yaml-optional)
+* [Optionale Dateien](#anker-files)
+* [Weitere nützliche Dateien](#anker-useful-files)
+* [Optionale Ordner](#anker-folders)
+* [Weitere nützliche Ordner](#anker-useful-folders)
+* [Software](#anker-software)
+* [Fragen und Antworten](#anker-faq)
+  * [MarkDown](#anker-faq-markdown)
+  * [Yaml](#anker-faq-yaml)
+
+
+<a name="anker-minimal"></a>
+## Minimale Erfordernisse
+Jedes AddOn hat einen eigenen Ordner mit einem einzigartigem und möglichst beschreibendem Namen.  
+  
+[@TODO:][SCREENSSHOT ORDNER]  
+  
+AddOns werden bei der Installation in Redaxo registriert, dazu sind einige Basisinformationen notwendig.
+Diese Basisinformationen werden in der Datei "package.yml" abgelegt. Prinzipiell sind mit dieser Datei
+alle Erfordernisse erfüllt, dass Redaxo ein AddOn installieren kann.  
```

Comment:
Vorschlag: …Erfordernisse erfüllt, damit ein AddOn in REDAXO installiert werden kann.